### ISSUE TITLE
Another weird _process_data error in console

### ISSUE THREAD

**Issue by Commaster** [TwitchIO/TwitchIO#315]
```
 Traceback (most recent call last):    
  File ".../python3.8/site-packages/twitchio/websocket.py", line 295, in _process_data              
    parsed = parser(data, self.nick)    
  File ".../python3.8/site-packages/twitchio/parse.py", line 61, in parser      
    elif groups[2] in {"PRIVMSG", "PRIVMSG(ECHO)"}:         
IndexError: list index out of range
```
https://github.com/TwitchIO/TwitchIO/blob/131e212ea48eeb82f5e7e9c6e35e776cb5fede4d/twitchio/websocket.py#L295
I feel like more info should be printed to understand why this is happening...

**Comment by Commaster** [comment]
yes, that is a problem. I'm just sitting in a chat channel, nothing happening and this comes up in the console, sometimes repeatedly.
Where is this try-except located so I could print the contents of the packet in addition to just the error message?

**Comment by chillymosh** [comment]
Are you able to update to python 3.9 / 3.10 and see if it still happens? This is not an error we've seen before.
Are you in non-English speaking channels? Has anything been typed in the channel at all?

**Comment by chillymosh** [comment]
We have found what causes this issue and will address it.
It's a very rare occurrence that just so happened to occur shortly after your previous issue, thankfully once someone else also reported this and noted it was the second time in 5 months it helped narrow down what it would be.

**Comment by IAmTomahawkx** [comment]
fixed in 95b8c17c70470b5d21560c29421edf7db7014614
